packages/mrml-core/src/mj_carousel/render.rs

```diff
@@ -246,10 +263,10 @@ impl<'root> Renderer<'root, MjCarousel, MjCarouselExtra> {
     }
 
     fn render_style(&self) -> Option<String> {
-        if self.element.children.is_empty() {
+        if self.get_children().is_empty() {
```

Comment:
We should create a dedicated function for that I guess

Reply:
For this, I've just checked the children count, as it's used a couple of lines down.